packages/create-llama/helpers/index.ts

```diff
@@ -174,18 +188,26 @@ export const installTemplate = async (
       vectorDb: props.vectorDb,
       model: props.model,
       framework: props.framework,
+      dataSource: props.dataSource,
     });
 
-    // Copy test pdf file
-    await copyTestData(
-      props.root,
-      props.framework,
-      props.packageManager,
-      props.engine,
-      props.openAiKey,
-      props.vectorDb,
-      props.contextFile,
-    );
+    if (props.engine === "context") {
+      if (
+        props.dataSource?.type === "file" &&
+        "contextFile" in props.dataSource.config
+      ) {
+        await copyContextData(props.root, props.dataSource.config.contextFile);
+      } else {
+        await copyContextData(props.root);
+      }
```

Comment:
@leehuwuj not a big think (for next time):
probably easier to read if you deal with this logic in `copyContextData`:

```suggestion
      await copyContextData(props.root, props.dataSource);
```